doc/governance.rst

```diff
@@ -20,95 +20,107 @@ the project community.
-Core developers are expected to review code
-contributions, can merge approved pull requests, can cast votes for and against
-merging a pull-request, and can be involved in deciding major changes to the
-API.
-
-New core developers can be nominated by any existing core developers. Once they
-have been nominated, there will be a vote by the current core developers.
-Voting on new core developers is one of the few activities that takes place on
-the project's private management list. While it is expected that most votes
-will be unanimous, a two-thirds majority of the cast votes is enough. The vote
-needs to be open for at least 1 week.
-
-Core developers that have not contributed to the project (commits or GitHub
-comments) in the past 12 months will be asked if they want to become emeritus
-core developers and recant their commit and voting rights until they become
-active again. The list of core developers, active and emeritus (with dates at
-which they became active) is public on the scikit-learn website.
+care. Being a maintainer allows contributors to more easily carry on with their
+project related activities by giving them direct access to the project's
+repository. Maintainers are expected to review code contributions, can merge
+approved pull requests, can cast votes for and against merging a pull-request,
+and are usually involved in deciding major changes to the API.
 
 Technical Committee
 -------------------
-The Technical Committee (TC) members are core developers who have additional
-responsibilities to ensure the smooth running of the project. TC members are expected to
-participate in strategic planning, and approve changes to the governance model.
-The purpose of the TC is to ensure a smooth progress from the big-picture
-perspective. Indeed changes that impact the full project require a synthetic
-analysis and a consensus that is both explicit and informed. In cases that the
-core developer community (which includes the TC members) fails to reach such a
-consensus in the required time frame, the TC is the entity to resolve the
-issue.
-Membership of the TC is by nomination by a core developer. A nomination will
-result in discussion which cannot take more than a month and then a vote by
-the core developers which will stay open for a week. TC membership votes are
-subject to a two-third majority of all cast votes as well as a simple majority
-approval of all the current TC members. TC members who do not actively engage
-with the TC duties are expected to resign.
+
+The Technical Committee (TC) members are maintainers who have additional
+responsibilities to ensure the smooth running of the project. TC members are
+expected to participate in strategic planning, and approve changes to the
+governance model. The purpose of the TC is to ensure a smooth progress from the
+big-picture perspective. Indeed changes that impact the full project require a
+synthetic analysis and a consensus that is both explicit and informed. In cases
```

Comment:
I don't understand the "Indeed, changes that ... synthetic analysis ... and informed." sentence. Why Indeed? What is a synthetic analysis? Maybe we can reformulate it? The sentence was already like this in the old version so also Ok to punt.

Reply:
That sentence makes sense to me lol, yet happy to rephrase. But as you say, I'm not changing it here, it's the old phrasing.

Reply:
To me it sounds like complete gibberish or something written by a marketing company that stuffed all the buzzwords into one sentence :D

Reply:
Let's talk about it when we talk about the TC -> SC issues in a different PR.